components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -79,6 +80,8 @@
 import java.util.Map;
 import java.util.Set;
 
+import static org.wso2.carbon.user.core.UserCoreConstants.RealmConfig.PROPERTY_DOMAIN_NAME;
```

Comment:
since all the other constants of this class are not static imports, shall we change this also for consistency?

Reply:
Fixed: f999948935ff2081b9362c4cbe0d6e8f5ee238db